### ISSUE TITLE
Bold text and big borders

### ISSUE THREAD

**Issue by Bleznudd** [solus-project/evopop-gtk-theme#70]
Don't know if this is the expected appearance or a bug, but I've **bold** text in a lot of places where it shouldn't be. As example in my upper panel (I'm using **Budgie DE**) clock and battery percentage are bold, the same for applications names in the menu.
Further more windows borders are really big, and this is ugly in < 14" screens. How to modify the height of the titlebar?

If needed, here are some info about my system

- Manjaro Linux 17.0.6 Gellivara
- kernel linux 4.9.58-1
- 64 bit OS
- budgie-desktop  10.4+8+gff09a510-1

**Comment by poltertec** [comment]
screenshot?

**Comment by Bleznudd** [comment]
Sure.
Here's the comparison between Evopop Azure and Arc Darker (but the font is the same for every other GTK theme)


Budgie Menu:
(all names are bold with Evopop)
![menu-arc](https://user-images.githubusercontent.com/33193871/32147496-7d0a1228-bce8-11e7-8cbb-205fe5901b2e.png)
![menu](https://user-images.githubusercontent.com/33193871/32147500-7f357f4c-bce8-11e7-9437-9cb7f30d7285.png)

Budgie Panel:
(clock and battery percentage are bold)
![panel-arc](https://user-images.githubusercontent.com/33193871/32147497-7d993b7e-bce8-11e7-8cc4-5231d6a3046f.png)
![panel](https://user-images.githubusercontent.com/33193871/32147501-7fbb38ee-bce8-11e7-842f-494e39814aef.png)

Budgie Setting:
(Window title and Selectable items are bold)
![settings-arc](https://user-images.githubusercontent.com/33193871/32147498-7e21e53c-bce8-11e7-9c91-73696f8c6843.png)
![settings](https://user-images.githubusercontent.com/33193871/32147499-7eca6a40-bce8-11e7-865d-c0c3d4f31628.png)

The same differences happens with other programs and GUI elements, these are only some examples

**Comment by poltertec** [comment]
I'll look into it.
I can understand your problem with a <14 screen, however already tried non bold clock some time ago.
Looked ugly on >24 screen with high resolution, so that will not change.

#71

**Comment by poltertec** [comment]
In latest commit budgie menu font weight was set to normal.
Also if 'Stylize Regions' aren't selected clock and battery percentage shouldn't be bold since this commit.

**Comment by Bleznudd** [comment]
Thanks for the menu font weight.
About the problem with <14 inch screens, it involves the height of the window borders, not the weight of the font.

Here you can see the height of the border of gnome terminal with Materia Compact theme

![materia-compact-terminal](https://user-images.githubusercontent.com/33193871/32405235-c541ea0c-c161-11e7-998f-034f79512f2c.png)

and here you can see the height of the same border with EvoPop Azure theme

![evopop-terminal](https://user-images.githubusercontent.com/33193871/32405234-c4a69e26-c161-11e7-9c18-6ddd1fda1523.png)

The second one is around 10px thicker than the first.
Assuming that you're working on a 10" low res pc, small window borders means more space for content.
Obviusly, bigger is the screen less relevant the height of borders is, but if is possibile to shrink a bit this parameter, I would ask you to point me out which files I have to modify.
Thanks in advance

**Comment by poltertec** [comment]
Yes I know the height should be dropped by around 6px already in last
commit.

Op 4 nov. 2017 13:46 schreef "Bleznudd" <notifications@github.com>:

**Comment by poltertec** [comment]
Tweaked it some more.
Could you check if this is pleasant for your monitor?
